pyaedt/edb_core/layout.py

```diff
@@ -1112,6 +1112,8 @@ def unite_polygons_on_layer(self, layer_name=None, delete_padstack_gemometries=F
             Layer Name on which unite objects. If ``None``, all layers will be taken.
         delete_padstack_gemometries : bool, optional
             ``True`` to delete all padstack geometry.
+        net_list : list[str] : optional
+            Net list filter, If ``None``, all nets will be taken.
```

Comment:
```suggestion
            Net list filter. The default is ``[]``, in which case all nets are taken.
```
Is this a list of filters or a list of filters to exclude from X? I find the current description unclear.